pkg/api/component_report.go

```diff
@@ -119,6 +119,7 @@ func GetComponentReportFromBigQuery(client *bqcachedclient.Client, gcsBucket str
 	cacheOption cache.RequestOptions,
 ) (apitype.ComponentReport, []error) {
 	generator := componentReportGenerator{
+		CacheType:     "data",
```

Comment:
What does CacheType do? I don't see it used anywhere

Reply:
It is used in the cacheKey, added a comment [here](https://github.com/openshift/sippy/blob/9dfe15eea256c6a54d72bca9d0a56b0c532e319c/pkg/api/utils.go#L47) but worth noting in the struct as well

Reply:
What's it's purpose though? I'm not seeing anything read from it

Reply:
I see - `CacheType: data` doesn't really mean anything then, it's just used to force a different cache key.  What about a version field? Maybe an integer `GeneratorVersion` that can bumped as needed if we make incompatible changes again

Reply:
Thought was if the same generator cached different types of objects, data vs. report that there was meaning.  But I don't mind switching it to version either, that does fit better for the current usage.